src/main/webapp/resources/js/components/sequence-files/SingleEndFileRenderer.jsx

```diff
@@ -27,6 +28,7 @@ export function SingleEndFileRenderer({
   removeSampleFiles = () => {},
   getProcessingState = () => {},
   qcEntryTranslations,
+  displayConcatenationCheckbox,
```

Comment:
If you are setting defaults for the function you should also probably set defaults for these as well, I would assume `false`

Reply:
Good catch..thanks!

Reply:
Set default in a0030449ca44d14e86faa4e3ece5b118bf115cc1